rapid_response_xblock/static/js/src/rapid.js

```diff
@@ -117,13 +172,32 @@
 
           var tspanText = tspan.text();
           tspan.text(tspanText + " " + word);
-          if (tspan.node().getComputedTextLength() > barWidth) {
+          if (tspan.node().getComputedTextLength() > hypotenuse) {
             // The new word would go beyond the bar width boundary,
             // so change tspan back to its old text and create one with the
             // new word on a new line.
+
```

Comment:
Since we're adding the loop-terminating logic below, I think we should cut off the above comment after 'boundary', then move the next two lines (after 'so') to line 198 where the described logic exists